plugin/src/test/java/io/jenkins/plugins/casc/SchemaGenerationTest.java

```diff
@@ -57,4 +59,12 @@ public void invalidSchemaShouldNotSucceed() throws Exception {
             ve.printStackTrace();
         }
     }
+
+    @Test
+    public void testRetrieveDocStringFromAttribute() {
+        String expectedDocString = "If checked, this will allow users who are not authenticated to access Jenkins in a read-only mode.";
+        String actualDocString = retrieveDocStringFromAttribute(hudson.security.FullControlOnceLoggedInAuthorizationStrategy.class, "allowAnonymousRead");
```

Comment:
This test doesn't belong here, it's testing retrieving docs works fine, that should be tested in a `ConfigurationAsCodeTest` (as the code is there)

what should be tested here is the HTML sanitisation, or the sanitising logic should be moved to a separate more specific class

Reply:
@timja I seem to have got confused isn't the html sanitisation been checked here since the actual string does not have any tags ?

Reply:
Nothing in that test proved html sanitisation, you could bump the version of Jenkins and someone may have removed the html and the test would still pass

Reply:
Yeah I will modify this test and move the original test to ConfigurationAsCodeTest.